cmd/earth/main.go

```diff
@@ -955,7 +1039,7 @@ func (app *earthApp) actionOrgCreate(c *cli.Context) error {
 		return errors.New("invalid number of arguments provided")
 	}
 	org := c.Args().Get(0)
-	sc := secretsclient.NewClient(app.apiServer, app.sshAuthSock, app.publicKey, app.console.Warnf)
+	sc, _ := secretsclient.NewClient(app.apiServer, app.sshAuthSock, app.console.Warnf)
```

Comment:
Add a comment about why error is ignored?

Reply:
errors are now correctly handled.